pkgs/games/tuxtype/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, pkgconfig, librsvg, SDL, SDL_image, SDL_mixer, SDL_ttf }:
+
+stdenv.mkDerivation rec {
+  version = "1.8.3";
+  name = "tuxtype-${version}";
+
+  src = fetchurl {
```

Comment:
You should use `fetchFromGitHub` with `rev = "upstream/${version}"`.

Reply:
Weird... the source code is different when i do this.

  src = fetchFromGitHub {
    owner = "tux4kids";
    repo = "tuxtype";
    rev = "upstream/${version}";
    sha256 = "1i33rhi9gpzfml4hd73s18h6p2s8zcr26va2vwf2pqqd9fhdwpsg";
  };

substitute(): ERROR: file 'Makefile.in' does not exist
builder for '/nix/store/y26mjr7nfg5if9k06k77zai964gd8xmr-tuxtype-1.8.3.drv' failed with exit code 1

Makefile.in existed when I used fetchurl but not when I use fetchFromGitHub.

Reply:
Are you sure that you used the correct hash before? `fetchFromGitHub` will resolve to the same URL.

Reply:
`Makefile.in` would not be in the VCS repository. It is generated by `autoconf` from `Makefile.am` (add `autoreconfHook` to `nativeBuildInputs`).

Reply:
Thanks for all your help on this.